cli/tools/vendor/build.rs

```diff
@@ -68,6 +68,14 @@ pub fn build(
   graph.lock()?;
   graph.valid()?;
```

Comment:
Removing this in https://github.com/denoland/deno/pull/15390/files#diff-25670ef0ccc50ba364743d0195b7ed54396876142b482fa36b99926bc1d6b46bL69 because it would error on some bare specifiers in type declarations, which wouldn't occur with `deno run` even when doing `--check=remote`.